src/test/suite/fixtures/recorded/languages/rust/changeState.yml

```diff
@@ -0,0 +1,28 @@
+languageId: rust
+command:
+  version: 1
+  spokenForm: change state
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: statement, includeSiblings: false}
+initialState:
+  documentContents: |-
+    let expensive_closure = |num| {
+        println!("calculating slowly...");
```

Comment:
Worth testing that "change state" with cursor inside `calculating` clears this line exactly